book.rb

```diff
@@ -0,0 +1,11 @@
+class Book
+  attr_accessor :title, :author
+
+  has_many :rentals
+  has_many :persons, through: :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+  end
```

Comment:
- [ ] A method for adding rentals should be present. You can read more on how to do that [here](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md)